app/src/main/java/com/github/se/signify/ui/Utils.kt

```diff
@@ -310,7 +308,7 @@ fun ScreenColumn(
               .padding(padding)
               .padding(16.dp)
               .verticalScroll(rememberScrollState())
-              .background(backgroundColor)
+              .background(MaterialTheme.colorScheme.background)
```

Comment:
Last sprint, my task was to refactor all color usage to rely on the theme, but I missed this one (hopefully not many others). I see you’ve removed the background color parameter, which makes sense since a screen should always use the background color.